cli/show_utils.go

```diff
@@ -90,3 +92,15 @@ func convertTimezone(ts time.Time) time.Time {
 		return ts
 	}
 }
+
+func formatCompressionRatio(original, compressed int64) string {
```

Comment:
nit: this is really space savings or compression percentage. Compression ratio would be `original/compressed`, sometimes noted as `(original/compressed):1`

https://en.wikipedia.org/wiki/Data_compression_ratio